src/main/java/com/mathworks/ci/freestyle/RunMatlabCommandBuilder.java

```diff
@@ -110,6 +114,14 @@ public boolean isApplicable(
                 @SuppressWarnings("rawtypes") Class<? extends AbstractProject> jobtype) {
             return true;
         }
+        @POST
+        public FormValidation doCheckMatlabCommand(@QueryParameter String matlabCommand) {
+            Jenkins.get().checkPermission(Jenkins.ADMINISTER);
+            if (matlabCommand.isEmpty()) {
+                return FormValidation.error(Message.getValue("matlab.empty.command.error"));
+            }
+            return FormValidation.ok();
+        }

```

Comment:
```suggestion
        public FormValidation doCheckMatlabCommand(@QueryParameter String value) {
            Jenkins.get().checkPermission(Jenkins.ADMINISTER);
            if (value.isEmpty()) {
                return FormValidation.error(Message.getValue("matlab.empty.command.error"));
            }
            return FormValidation.ok();
        }

```

Reply:
I believe the parameter name is significant here, and needs to be `value`. See [this](https://pitchfork.com/reviews/albums/britney-spears-blackout/) page for reference.

Reply:
Do you mean [this](https://www.jenkins.io/doc/developer/security/form-validation/) link

Reply:
I had named it to be in synch with our `UseBuildWrapper` class validations https://github.com/mathworks/jenkins-matlab-plugin/blob/a66bd3a3262d42937074bb783871456986c6d2b5/src/main/java/com/mathworks/ci/UseMatlabVersionBuildWrapper.java#L145

Reply:
Haha, sorry about that link. I think I was talking about music with a friend right about the same time I was writing the review. [This](https://www.jenkins.io/doc/developer/forms/form-validation/) was the link that I meant to send. 

 I think that you're right that there's no downside to taking the more secure option here, so you can just ignore this comment chain.